mlir-sycl/test/Transforms/inliner.mlir

```diff
@@ -142,3 +142,61 @@ gpu.func @gpu_func_callee() -> i32 attributes {passthrough = ["alwaysinline"]} {
 }
 
 }
+
+// -----
+
+// COM: Ensure functions in a SCC are fully inlined (requires multiple inlining iterations). 
```

Comment:
This test case doesn't always rely on multiple inlining iterations. It depends on which node in the SCC being visit first.
If you move `@callee` and `@always_inline_callee` before the other functions, then one iteration is enough.

Reply:
Assuming we want to rely on the SCC traversal order, you can simplify this test case by removing `@inline_hint_callee1` and `@always_inline_callee`, would still need 2 iterations.

Reply:
yes reduced the test